### ISSUE TITLE
Committee page is missing a `h2`

### ISSUE THREAD

**Issue by joshkimux** [AIGAdc/dcdesignweek.org-11ty#152]
## Description

Currently the committee page is missing a `h2` between the `h1` and the committee team names.

<img width="1792" alt="Screen Shot 2022-09-26 at 8 36 33 PM" src="https://user-images.githubusercontent.com/14154792/192405012-d622ed2e-1e3d-4018-94fb-54be781c5fdf.png">

## Recommendation

Adding a `h2` above the committee names titled "Teams" should do the trick! It can also allow us to remove "team" from each of the committee names 😄 (@silviascript for awareness and/or curiosity)

<img width="1679" alt="Screen Shot 2022-09-26 at 8 38 42 PM" src="https://user-images.githubusercontent.com/14154792/192405133-7c9693b6-8245-4d00-b070-37df5741f3b7.png">
